templates/homepage/index.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'homepage/base.html'%}
+{% load static %}
+{% load i18n %}
+
+{% block cssblock %}
+<link rel="shortcut icon" href="{% static 'homepage/images/favicon.png' %}" type="">
+
+<title> HyNiu Food & Drink </title>
+  <!-- bootstrap core css -->
+  <link rel="stylesheet" type="text/css" href="{% static 'homepage/css/bootstrap.css' %}" />
+  <!-- Custom styles for this template -->
+  <link href="{% static 'homepage/css/style.css' %}" rel="stylesheet" />
+  <!-- responsive style -->
+  <link href="{% static 'homepage/css/responsive.css' %}" rel="stylesheet" />
+    {% endblock %}
```

Comment:
Check indentation trong file cho chuẩn e nhé, cho code sáng sủa
```suggestion
{% endblock %}
```